src/beacon-api/app/routes/sites.py

```
@@ -275,59 +314,123 @@
+                "longitude": site.longitude
             }
         }
     }
 
 
-@router.get("/regions/list")
-async def get_regions(
+@router.get("/cities/list")
+async def get_cities(
     *,
     db: Session = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)